Jenkinsfile.integration

```diff
@@ -36,6 +36,15 @@ def cleanUp() {
 
 node('node-06') {
 	lock(resource: "core-integration-tests", inversePrecedence: true) {
+
+		properties([
+			parameters([
+				string(name: 'JENKINS_PROFILE', defaultValue: 'jenkins', description: 'To build cache dependencies and run slow tests, change this value to jenkins-extensive.', ),
```

Comment:
This parameter did not exist before and does not seem to relate to #1577; is it really necessary?

Reply:
That's a good point, I am gonna remove it from `integration`, do we need the `JENKINS_PROFILE` from `core`, why exactly `JENKINS_PROFILE` env variable does in the core?

Reply:
IIRC it is used to include/exclude some long running tests.